### PR TITLE
Fix line items

### PR DESCRIPTION
**Pull request by mmiszy**
Shipping costs were not included in line items. This caused some payment methods to fail because total of all line items was less than total that we tried to charge/authorize.

### CODE REVIEW

File: src/modules/adyen/adyen-api.ts [hunk 1]
```diff
@@ -51,46 +51,71 @@ export async function paymentGatewayInitializeSessionEventToAdyen(
...
+            shippingPrice.net.amount,
+            shippingPrice.net.currency,
+          ),
+          amountIncludingTax: getAdyenIntegerAmountFromSaleor(
+            shippingPrice.gross.amount,
+            shippingPrice.gross.currency,
+          ),
+          id: deliveryMethod.id,
+        }
+      : null;
```
Comment (mmiszy) [inline src/modules/adyen/adyen-api.ts:79]: I'll add a test for this.

**Review by bmigirl — APPROVED** [review]

File: src/modules/webhooks/__tests__/utils.ts [hunk 2]
```diff
@@ -88,16 +88,35 @@ export const createMockTransactionChargeRequestedEvent: CreateMockAsyncFn<
         authorizedAmount: {
           __typename: "Money",
           currency: "PLN",
-          amount: 99.99,
+          amount: 99.99 + 123.0,
```
Comment (witoszekdev) [inline src/modules/webhooks/__tests__/utils.ts:91]:
```suggestion
          amount: 99.99 + 123.0, // gross delivery cost
```

File: src/modules/webhooks/__tests__/utils.ts [hunk 3]
```diff
@@ -78,7 +78,7 @@ export const createMockTransactionChargeRequestedEvent: CreateMockAsyncFn<
       recipient: await createMockApp(overrides?.recipient),
       action: {
         __typename: "TransactionAction",
-        amount: 99.99,
+        amount: 99.99 + 123.0,
```
Comment (witoszekdev) [inline src/modules/webhooks/__tests__/utils.ts:81]:
```suggestion
        amount: 99.99 + 123.0, // gross delivery cost
```

File: src/modules/webhooks/transaction-initialize-session.test.ts [hunk 4]
```diff
@@ -389,5 +390,72 @@ describe(`TransactionInitializeSessionWebhookHandler`, () => {
...
+
+    it("should send line items if amount is total", async () => {
+      const spy = vi.spyOn(AdyenCheckoutAPI.prototype, "payments");
+
+      const event = await createMockTransactionInitializeSessionEvent({
+        data: createMockAdyenDataActionNotRequired(),
+        sourceObject: getSourceObject(),
+      });
+      await TransactionInitializeSessionWebhookHandler(event, testEnv.TEST_SALEOR_API_URL);
+      console.log(spy.mock.lastCall?.[0].lineItems);
```
Comment (witoszekdev) [inline src/modules/webhooks/transaction-initialize-session.test.ts:402]:
```suggestion
```

**Review by bmigirl — APPROVED** [review]

**Pull request merged**